posts/2021-01-29-airports-at-scale.adoc

```diff
@@ -0,0 +1,39 @@
+= Airports at Scale
+nMoncho
+v1.0, 2020-01-29
+:title: Airports at Scale
+:tags: [database, nosql, big data, modelling, programming]
+
+== Assessing NoSQL Databases
+
+In a https://blog.lunatech.com/2018/11/lunatech-airport-assessment/[previous blog post], I explained the assessment we were using at Lunatech, detailing a solution using a common tech stack, Java with Spring Boot and PostgreSQL. An idea mentioned was that it would’ve been possible to also develop a solution using a NoSQL database, since we did receive such solutions, this got me thinking: could every NoSQL family easily solve each requirement? What would be the advantages of using them over SQL? And, more importantly, do they fit in the problem and domain?
```

Comment:
```suggestion
In a https://blog.lunatech.com/2018/11/lunatech-airport-assessment/[previous blog post], I explained the assessment we were using at Lunatech, detailing a solution using a common tech stack, Java with Spring Boot and PostgreSQL. An idea mentioned was that it would’ve been possible to also develop a solution using a NoSQL database, and since we did receive such solutions, this got me thinking: could every NoSQL family easily solve each requirement? What would be the advantages of using them over SQL? More importantly, do they fit in the problem and domain?
```